compiler/scenario-service/protos/scenario_service.proto

```diff
@@ -418,7 +418,7 @@ message FailedAuthorization {
     Identifier template_id = 1;
     string choice_id = 2;
     Location location = 3; // optional
-    repeated Party controllers = 4;
+    reserved 4; // repeated Party controllers
```

Comment:
```suggestion
```
As far I am concerned, we do not need any backward compatibility for the scenario service proto. 
It is just better to drop the field completely. 
cc @moritzkiefer-da

Reply:
absolutely, let’s not accumulate legacy without any good reason.